### PR TITLE
Members find all test

### PR DESCRIPTION
**Pull request by xiomaraOcampo**
test and documentacion of Member FindAll

### CODE REVIEW

File: test/testimonial.Test.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ describe('#GET /testimonials', async function () {
     expected: {
       errors: [
         {
-          "msg": "Testimonial with page 22 doesn't exists."
+          "msg": "Testimonial with page 22 doesn't exist."
```
Comment (FacuBar) [inline test/testimonial.Test.js:38]:
```  
3) Login users
       unregistered user:

      AssertionError: expected { Object (errors) } to deeply equal { Object (errors) }
      + expected - actual

       {
         "errors": [
           {
      -      "msg": "User with email not@registered.com doesn't exist."
      +      "msg": "User with email not@registered.com doesn't exists."
           }
         ]
       }

      at Context.<anonymous> (test/user.Test.js:187:32)
      at processTicksAndRejections (node:internal/process/task_queues:96:5)
 ```

**Review by FacuBar — COMMENTED** [review]
```
  2) #GET /members
       if the page of Member does not exist  returns an  error message:
     Error: expected 404 "Not Found", got 400 "Bad Request"
      at Context.<anonymous> (test/member.Test.js:53:8)
      at processImmediate (node:internal/timers:464:21)
  ----
      at Test._assertStatus (node_modules/supertest/lib/test.js:252:14)
      at /home/efacubar/Documents/workspaces/javascript/OT125-SERVER/node_modules/supertest/lib/test.js:306:17
      at Test._assertFunction (node_modules/supertest/lib/test.js:285:13)
      at Test.assert (node_modules/supertest/lib/test.js:164:23)
      at Server.localAssert (node_modules/supertest/lib/test.js:120:14)
      at Object.onceWrapper (node:events:509:28)
      at Server.emit (node:events:390:28)
      at Server.emit (node:domain:475:12)
      at emitCloseNT (node:net:1681:8)
      at processTicksAndRejections (node:internal/process/task_queues:82:21)

  3) #POST /members
       The name of member must not be repeated:
     Error: expected 409 "Conflict", got 200 "OK"
      at Context.<anonymous> (test/member.Test.js:127:8)
      at processImmediate (node:internal/timers:464:21)
  ----
      at Test._assertStatus (node_modules/supertest/lib/test.js:252:14)
      at /home/efacubar/Documents/workspaces/javascript/OT125-SERVER/node_modules/supertest/lib/test.js:306:17
      at Test._assertFunction (node_modules/supertest/lib/test.js:285:13)
      at Test.assert (node_modules/supertest/lib/test.js:164:23)
      at Server.localAssert (node_modules/supertest/lib/test.js:120:14)
      at Object.onceWrapper (node:events:509:28)
      at Server.emit (node:events:390:28)
      at Server.emit (node:domain:475:12)
      at emitCloseNT (node:net:1681:8)
      at processTicksAndRejections (node:internal/process/task_queues:82:21)
```

**Comment by xiomaraOcampo** [comment]
Done.

**Review by thadek — APPROVED** [review]
image
All ok!

**Review by FacuBar — APPROVED** [review]

**Review by RominaSaravia — APPROVED** [review]
Falta resolver conflictos. 
Pero esta aprobado

**Pull request merged**